src/gc/gc.cpp

```diff
@@ -26608,7 +26608,7 @@ void gc_heap::revisit_written_pages (BOOL concurrent_p, BOOL reset_only_p)
                             card_table [card_word (card_of (background_written_addresses [i]))] = ~0u;
                             dprintf (3,("Set Cards [%p:%p, %p:%p[",
                                         card_of (background_written_addresses [i]), g_addresses [i],
-                                        card_of (background_written_addresses [i]+OS_PAGE_SIZE), background_written_addresses [i]+OS_PAGE_SIZE));
+                                        card_of (background_written_addresses [i]+GC_PAGE_SIZE), background_written_addresses [i]+GC_PAGE_SIZE));
     #endif //NO_WRITE_BARRIER
                             uint8_t* page = (uint8_t*)background_written_addresses[i];
                             dprintf (3, ("looking at page %d at %Ix(h: %Ix)", i, 
```

Comment:
I think there may need to be more places to flip from `OS_PAGE_SIZE` to `GC_PAGE_SIZE` in the GC, for performance reasons. Could you please open an issue on it under `area-GC`?